packages/lib-classifier/src/store/Subject.spec.js

```diff
@@ -40,60 +78,17 @@ describe('Model > Subject', function () {
     })
 
     it('should call the onToggleFavourite callback', function () {
-      expect(subject.onToggleFavourite).to.have.been.calledOnceWith(subject.id, subject.favorite)
+      expect(subject.onToggleFavourite.withArgs(subject.id, subject.favorite)).to.have.been.calledOnce()
     })
   })
 
-  describe('addToCollection', function () {
+  describe('Actions > addToCollection', function () {
     before(function () {
       subject.addToCollection()
     })
 
     it('should call the onAddToCollection callback', function () {
-      expect(subject.onAddToCollection).to.have.been.calledOnceWith(subject.id)
-    })
-  })
-
-  describe('openInTalk', function () {
```

Comment:
I don't think these tests for going to Talk should be removed. Looks like I started them but forgot to copy the expectations across when I moved the code. https://github.com/zooniverse/front-end-monorepo/pull/901

Reply:
I've added these tests in #1031.

Reply:
Thanks for catching this. When I glanced at it, I didn't realize that this was intended to test something different from `openTalkPage` in the `SubjectStore` spec file and then thought it was cruft.

Reply:
I thought I'd tested these too, but I don't see any tests in the PR that added them.